packages/documentation/src/pages/forms/form-tutorial-basic.mdx

```diff
@@ -80,7 +87,7 @@ const formConfig = {
 };
 ```
 
-There's a lot of information already there, and you can check out the [form config specifications](https://github.com/usds/us-forms-system/tree/master/docs) to see what each property means. For now, we're going to look at the content of the form, which lives in `chapters`.
+There's a lot of information already there, and you can check out the [form config specification](/forms/docs/spec) to see what each property means. For now, we're going to look at the content of the form, which lives in `chapters`.
```

Comment:
This doesn't exist, but we'll need to create it as part of moving the usfs docs over.